apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsLinkArrayControl.tsx

```diff
@@ -80,13 +88,29 @@ const EditLinkItem = ({
     uischema,
   )
 
+  // Disable scrolling on parent container when editing a link item, as this
+  // is an absolutely-positioned overlay
+  useEffect(() => {
+    const parent = document.getElementById(FORM_BUILDER_PARENT_ID)
+    if (parent) {
+      parent.scrollTop = 0
+      parent.style.overflow = "hidden"
+    }
+
+    return () => {
+      if (parent) {
+        parent.style.overflow = "auto"
```

Comment:
should we use jotai to make a global reference instead of doing `document.getElementById`?